Source/WebKit/UIProcess/WebPageProxy.h

```diff
@@ -2862,7 +2863,7 @@ class WebPageProxy final : public API::ObjectImpl<API::Object::Type::Page>, publ
     bool toolbarsAreVisible() const { return m_toolbarsAreVisible; }
     void setToolbarsAreVisible(bool);
 
-#if USE(GBM) && (PLATFORM(GTK) || PLATFORM(WPE))
+#if (USE(GBM) || OS(ANDROID)) && (PLATFORM(GTK) || PLATFORM(WPE))
```

Comment:
Totally minor, but would it make sense to use the same order consistently to facilitate future searches?


```suggestion
#if (PLATFORM(GTK) || PLATFORM(WPE)) && (USE(GBM) || OS(ANDROID))
```

Like line 619